### PR TITLE
Added mute stream func

### PR DESCRIPTION
**Pull request by allocom**
A small addition, mute & unmute events included.

### CODE REVIEW

File: sound/soc/bcm/allo-katana-codec.c [hunk 1]
```diff
@@ -42,8 +42,10 @@
 #define KATANA_CODEC_DEEMPHASIS		(KATANA_CODEC_VIRT_BASE + 6)
 #define KATANA_CODEC_DOP			(KATANA_CODEC_VIRT_BASE + 7)
 #define KATANA_CODEC_FORMAT			(KATANA_CODEC_VIRT_BASE + 8)
-#define KATANA_CODEC_COMMAND		(KATANA_CODEC_VIRT_BASE + 9)
-#define KATANA_CODEC_MAX_REGISTER	(KATANA_CODEC_VIRT_BASE + 9)
+#define KATANA_CODEC_MUTE_STREAM	(KATANA_CODEC_VIRT_BASE + 9)
+#define KATANA_CODEC_COMMAND		(KATANA_CODEC_VIRT_BASE + 10)
```
Comment (pelwell) [inline sound/soc/bcm/allo-katana-codec.c]: It's your product, so you can do what you want, but by changing the offset of the COMMAND register aren't you creating a forwards- and backwards-compatibility problem? Why not put MUTE_STREAM at offset 10?

File: sound/soc/bcm/allo-katana-codec.c [hunk 2]
```diff
@@ -236,7 +242,22 @@ static int katana_codec_set_fmt(struct snd_soc_dai *dai, unsigned int fmt)
 	return 0;
 }
 
+int katana_codec_dai_mute_stream(struct snd_soc_dai *dai, int mute, int stream)
+{
+	struct snd_soc_codec *codec = dai->codec;
+	struct katana_codec_priv *katana_codec = snd_soc_codec_get_drvdata(codec);
```
Comment (pelwell) [inline sound/soc/bcm/allo-katana-codec.c]: Even though this is downstream code we try to stick to kernel coding conventions - please split this long line,

File: sound/soc/bcm/allo-katana-codec.c [hunk 3]
```diff
@@ -236,7 +242,22 @@ static int katana_codec_set_fmt(struct snd_soc_dai *dai, unsigned int fmt)
...
+	int ret = 0;
+
+	ret = regmap_write(katana_codec->regmap, KATANA_CODEC_MUTE_STREAM, mute);
```
Comment (pelwell) [inline sound/soc/bcm/allo-katana-codec.c]: Another long line.

**Review by pelwell — COMMENTED** [review]
Apart from the line comments, please use a proper email address for your commit ("Your Name" won't do), and  include a "Signed-off-by:" attribution.

**Comment by allocom** [comment]
Changed the position of KATANA_CODEC_MUTE_STREAM.  I had set my editors tab size as 4, so could not identify the line length.

**Comment by pelwell** [comment]
That's looking much better. Now you just need to tell git your name and email address:
```
$ git config --global user.name Jaikamur
$ git config --global user.email jaikamur@cem-solutions.com
```
Then redo the changes as a single commit:
```
$ git reset --soft 58eb1e1c
$ git commit -s
$ git push -f
```

**Comment by allocom** [comment]
For 58eb1e1c, it says "unknown revision or path".  Should I give first commit # ( 0a5dc0e ) ??

**Comment by pelwell** [comment]
Use the hash of the first commit that isn't one of yours.

**Comment by allocom** [comment]
Thanks.. Hope its right

**Comment by pelwell** [comment]
Close - you need to move your reset to one commit further backwards. Alternatively, `git fetch origin`, `git rebase` and use `git reset --soft 58eb1e1c`.

**Comment by allocom** [comment]
I did git fetch origin, git rebase, git reset --soft 0a5dc0e, git commit -s and git push -f.
but still 2 commits ??

**Comment by pelwell** [comment]
Yes, that is expected. Read my instruction carefully:

> Use the hash of the first commit _that isn't one of yours_.

**Comment by allocom** [comment]
Got a single commit. Hope its correct now. Thanks

**Comment by allocom** [comment]
Thanks Pelwell.

**Pull request merged**